ThingIFSDK/ThingIFSDK/CommandForm.swift

```diff
@@ -64,20 +64,39 @@ open class CommandForm<ConcreteAlias: Alias>: NSObject, NSCoding {
     }
 
     open func encode(with aCoder: NSCoder) {
-        aCoder.encode(self.actions, forKey: "actions")
+        var dictArray: [[String : Any]] = []
+        self.actions.forEach {
+            action in dictArray.append(
```

Comment:
@yongpingchen 

We can use non-trait and trait actions in unified form by introducing `TraitAlias` and `NonTraitAlias`.

`Stored actions` is following form in swift dictionary style.

`nonTraitAction`:
```swift
[
  [
    "alias" : NonTraitAlias(),
    "actions" : [
      "turnPower" : {
          "power" : true
      }
    ]
  ]
]
```

traitAction:
```swift
[
  [
    "alias" : TraitAlias("turnPowerAlias"),
    "actions" : [
      "turnPower" : {
          "power" : true
      }
    ]
  ],

  [
    "alias" : TraitAlias("setTemperatureAlias"),
    "actions" : [
      "setTemperature" : {
          "temperature" : 25
      }
    ]
  ]
]
```